cadence/contracts/GametvNFT.cdc

```diff
@@ -34,27 +36,24 @@ pub contract GametvNFT: NonFungibleToken {
             self.currentCount = count
         }
 
-        init(typeId: String, maxCount: UInt64, metadata: {String : AnyStruct}) {
-            pre {
-                metadata.length != 0: "metadata cannot be empty"
-            }
+        init(typeId: String, maxCount: UInt64) {
 
-            if (GametvNFT.nftTypes.keys.contains(typeId)) {
+            if (GametvNFT.activeNftTypes.keys.contains(typeId)) {
```

Comment:
Although this situation should never happen, you should check both active and historic dicts just to be safe.

Reply:
Then we won't be able to achieve what we want. We separated them bcoz we don't want to execute queries on historic data as it is too big.
I understand your point, I think I asked the same thing on thursday as well, when we decided to do this check at backend level

Reply:
Right, we don't want to use the historic data. This check is incomplete without it though.. let's just leave this as-is then.